clustered/client/src/main/java/org/ehcache/clustered/client/internal/service/DefaultClusteringService.java

```diff
@@ -132,12 +132,19 @@ public void start(final ServiceProvider<Service> serviceProvider) {
       try {
         entity = entityFactory.retrieve(entityIdentifier, configuration.getServerConfiguration());
       } catch (EntityNotFoundException e) {
-        /*
-         * If the connection failed because of a creation failure, re-throw the creation failure.
-         */
-        throw new IllegalStateException(failure == null ? e : failure);
+        if (failure == null) {
+          throw new IllegalStateException("The server-side storage manager does not exist."
```

Comment:
The entity is defined as _Ehcache clustered entity_ in our clustered cache doc. So I guess that's what we should be using here.